lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
```

Comment:
> instances can override electron versions at construction time

Ohhh, good point. Did not think about that at all. Still relevant for `.version` maybe?